packages/storefront-lwc/src/modules/commerce/checkout/checkout.html

```diff
@@ -0,0 +1,15 @@
+<template>
+    <commerce-context-heading header-text="Checkout"></commerce-context-heading>
+    <div class="container">
+        <div class="row justify-content-md-center">
+            <div class="col-md-8" if:false={onConfirmation}>
+                <commerce-checkout-shipping shipping={checkoutDetails.shipping} onshippingupdate={updateShippingDetails} stage={routeParams.checkoutStage}></commerce-checkout-shipping>
+                <commerce-checkout-payment payment={checkoutDetails.payment} oncheckoutupdate={updateCheckoutdetails} stage={routeParams.checkoutStage}></commerce-checkout-payment>
+                <lwce-link onclick={placeOrder} href={nextHref} class="btn btn-primary btn-block checkout-btn">{nextText}</lwce-link>
```

Comment:
Everything is encapsulated within this checkout component, but we still have routes for each stage. The `href` is calculated based on the current "stage".